cmd/plugin/cli/preflight.go

```diff
@@ -12,7 +12,13 @@ import (
 )
 
 const (
-	preflightSpec = "https://raw.githubusercontent.com/storageos/storageos.github.io/master/yaml/preflight.yaml"
+	// defaultCollectorImage can be removed once
+	// https://github.com/replicatedhq/troubleshoot/pull/392 merges, then it
+	// will default to "replicatedhq/troubleshoot:latest".
+	defaultCollectorImage = "storageos/troubleshoot:latest"
```

Comment:
Do they have versioned image? `latest` is not always the best solution.

Reply:
Yes, good idea to set a tagged release once there is one.  I'll update the tag to be the commit ref for now.